<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Balistic Model</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/img_main.png</normaloff>images/img_main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(187, 208, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор&lt;/p&gt;&lt;p&gt;планеты&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Earth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Земля</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Moon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Луна</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CustomPlanet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Польз. планета</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(99, 255, 96);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор&lt;/p&gt;&lt;p&gt;ракеты&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_RocketOne">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>№ I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_RocketTwo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>№ II</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CustomRocket">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(129, 218, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Польз. ракета</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Stop">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">     background-color: rgb(255, 79, 79);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
        </property>
        <property name="text">
         <string>Стоп</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Радиус Планеты [м]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Radius">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Введите значение радиуса...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Масса планеты [кг]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PlanetMass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Введите значение массы...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Тяга двигателей [н]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Trust">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Введите значение силы...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Расход топлива [кг/сек]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_FuelPerSecond">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Введите значение расхода...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Масса ракеты [кг]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_RocketMass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Введите значение массы...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_time1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">     background-color: rgb(174, 211, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_time2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">     background-color: rgb(174, 211, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_time3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">     background-color: rgb(174, 211, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
          </property>
          <property name="text">
           <string>&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_time4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">     background-color: rgb(174, 211, 255);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: rgb(126, 126, 126);
     font: bold 14px;
     padding: 6px;</string>
          </property>
          <property name="text">
           <string>&gt;&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 194, 192);
border-color: rgb(124, 124, 124);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="ActionOpenHelp"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 194, 192);</string>
   </property>
  </widget>
  <action name="ActionOpenHelp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть справку</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
